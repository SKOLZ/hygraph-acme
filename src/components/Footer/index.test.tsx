import { render } from "@testing-library/react"
import { Footer } from "."

describe("footer", () => {
  it("renders correctly", () => {
    const {container} = render(<Footer />);
    expect(container).toMatchSnapshot();
  })
})
