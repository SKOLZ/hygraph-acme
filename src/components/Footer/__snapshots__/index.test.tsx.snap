// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer renders correctly 1`] = `
<div>
  <footer
    class="mx-auto w-full max-w-screen-xl px-8"
  >
    <div
      class="text-5 flex flex-col items-center gap-12 border-t border-t-gray-200 py-8 text-center text-secondary-400 md:flex-row-reverse md:justify-between"
    >
      <div
        class="flex gap-4"
      >
        <a
          href="#"
        >
          Terms
        </a>
        <a
          href="#"
        >
          Privacy
        </a>
        <a
          href="#"
        >
          Cookies
        </a>
      </div>
      <p>
        Â© 2023 Acme Co. All rights reserved.
      </p>
    </div>
  </footer>
</div>
`;
